nemo/core/neural_types/elements.py

```diff
@@ -69,28 +70,35 @@ class ElementType(ABC):
     We are relying on Python for inheritance checking"""
 
     def __str__(self):
+        if torch.jit.is_scripting():
```

Comment:
How about just adding one init in this class and no need to put init in all subclasses ?

Reply:
Good suggestion, but unfortunately this is not supported in TorchScript(
Inability to call `super().method()` is a known limitation, still not fixed https://github.com/pytorch/pytorch/issues/42885

Reply:
Ok, no need to change then